src/convertToObject.js

```diff
@@ -6,7 +6,18 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const styleObject = {};
+  const styleCommands = sourceString.split(';');
+
+  styleCommands.forEach((element) => {
+    const [key, value] = element.split(':');
+
+    if (key && value) {
+      styleObject[key.trim()] = value.trim();
+    }
+  });
```

Comment:
According to the [CODE STYLE] checklist item, all chained methods called on the same level should have the same indentation. Although there are no chained methods here, consider that if chaining were to be used, proper indentation would be important for readability.